appium/webdriver/common/multi_action.py

```diff
@@ -33,7 +33,7 @@
 class MultiAction:
     """
     deprecated:: 2.0.0
-        Please use W3C actions instead: http://appium.io/docs/en/commands/interactions/actions/
+        Please use W3C actions instead: https://github.com/appium/python-client?tab=readme-ov-file#multiactiontouchaction-to-w3c-actions
```

Comment:
Or 'mobile:' extensions?

Reply:
Actual replacement is W3C actions for the touch actions APIs. The mobile extension is possible alternative (could be not equivalent behavior) thus I think actions api is sufficient as API documentation.
README includes the alternative as well though as part of migration guide.